fedimint-bin-tests/src/federation.rs

```diff
@@ -139,6 +152,21 @@ impl Federation {
         info!("Using {name} gateway", name = gw.ln.name());
         Ok(())
     }
+
+    pub async fn kill_server(&mut self, peer_id: usize) -> anyhow::Result<()> {
+        let members = &mut self.members;
+        let fedimintd = std::mem::replace(&mut members[peer_id], None);
+        drop(fedimintd);
```

Comment:
To kill the server, I'm dropping `Fedimintd`, which drops it's `ProcessHandle` and therefore killing the process.

Reply:
Sure, should I introduce a field `clone_cound: usize` inside of `Fedimintd` and assert that to be 0. Or is there any other (and better) way to do this, that I don't know of?

Reply:
https://doc.rust-lang.org/stable/std/sync/struct.Arc.html#method.try_unwrap